packages/iris-grid/src/IrisGridPartitionSelector.scss

```diff
@@ -18,8 +18,7 @@
   }
   .partition-button-group {
     display: flex;
-    border: 1px var(--dh-color-hr);
-    border-style: none solid;
+    border-right: 1px solid var(--dh-color-hr);
     padding: 0 $spacer-2;
```

Comment:
Need to change the `padding` too. There's too much gap on the left with this padding. Should just be `padding-right: $spacer-2`